code/datums/traits/good.dm

```diff
@@ -86,6 +109,23 @@
 
 
 
+/datum/quirk/photographer
+	name = "Photographer"
+	desc = "You know how to handle a camera, shortening the cooldown between each shot."
```

Comment:
change "cooldown" to "delay", keeps OOC out of IC a bit imo

Reply:
good call